examples/llm/yamls/hf_causal_lm/gpt2-small.yaml

```diff
@@ -5,7 +5,7 @@ max_seq_len: 2048
 global_seed: 17
 
 # Run Name
-run_name: gpt2-small
+run_name: null
```

Comment:
@abhi-mosaic You started the pattern. You must finish it. IF LEFT BLANK WILL BE READ FROM ENV VAR.